nuxeo-coldstorage-web/elements/nuxeo-delete-content-from-coldstorage-button.js

```diff
@@ -0,0 +1,123 @@
+/**
+ (C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+ Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Is this a public or private code? For public code that's not the Copyright first line we use. For private code it's not under the Apache License.